### PR TITLE
submitting to basecamp marketplace

### PR DESCRIPTION
**Pull request by TheEnigmaticT**
Adding in for Unito per Xenia

### CODE REVIEW

**Comment by TheEnigmaticT** [comment]
@javan I'm not Xenia, so you may not be expecting this. 😛

File: integrations.yml [hunk 1]
```diff
@@ -11,6 +11,12 @@
 # 8 - Contracts and Proposals
 # 9 - Developer-only Tools
 #
+-
+  name: “Unito.io To-Do Sync”
+  description: “Sync your to do lists with other project management apps. Connect Basecamp to Trello, Asana, JIRA, Wrike, Planner, GitHub, GitLab, Bitbucket and more! Use Unito’s powerful filters to 2-way sync all of your to-do’s or just a few. Try it free today.”
+  website: “https://unito.io/?utm_source=basecamp&utm_medium=marketplace&utm_campaign=”
+  icon: “unito_logo.png”
```
Comment (javan) [inline integrations.yml]: Please use standard quotation marks. `"` instead of `“`.

**Comment by TheEnigmaticT** [comment]
Son of a b.

I'll fix and resubmit tomorrow.

**Comment by TheEnigmaticT** [comment]
Hey @javan Here's the fixed version of the file. Let me know if we need to do anything else! 😄

**Pull request merged**